app/src/main/java/com/example/bigowlapp/activity/HomePageActivity.java

```diff
@@ -188,6 +199,11 @@ public void setHomePageViewModel(HomePageViewModel homePageViewModel) {
         this.homePageViewModel = homePageViewModel;
     }
 
+    @VisibleForTesting
+    public void setSupervisorSchedulesAlarmManager(SupervisorSchedulesAlarmManager supervisorSchedulesAlarmManager) {
+        this.supervisorSchedulesAlarmManager = supervisorSchedulesAlarmManager;
+    }
+
```

Comment:
This method is not being used.

Refactor: Remove dead code.

Reply:
ok